semantic_release/cli.py

```diff
@@ -86,7 +86,9 @@ def version(**kwargs):
         return True
 
     set_new_version(new_version)
-    if config.get('semantic_release', 'version_source') == 'commit':
+    if config.get('semantic_release', 'commit_changes', fallback=(
```

Comment:
Will commit_changes set to false and version_source set to commit make it commit the changes? Also maybe commit_version_number is a better name?

Reply:
In that case, it would not commit the change. Setting `commit_changes` overloads other variables. I did it that way in order to keep the current behavior for everyone not interested in changing it. This way, in your example, the source is still defined by `version_source`, but it does not mean that the changes to the version file are commited. I don't know why someone would want to do that, but I think that if we allow people to get from vcs and write in file, we should also allow them to get from the file and not write in this file.

I will change the name to `commit_version_number` if you think it's clearer.

I will also include the change I asked you about in history.logs.evaluate_version_bump at line 44.

Reply:
Sounds good 👍